linkis-engineconn-plugins/elasticsearch/src/main/java/org/apache/linkis/engineplugin/elasticsearch/executor/client/EsClientFactory.java

```diff
@@ -192,6 +192,9 @@ private static HttpHost[] getCluster(String clusterStr) {
           .map(
               value -> {
                 String[] arr = value.replace("http://", "").split(":");
+                if (arr.length < 2) {
+                  return new HttpHost("127.0.0.1", 9200);
```

Comment:
this should use ```new HttpHost(arr[0].trim(), 80);```, and add length <1, throw error; 

http without port, the default port is 80.   https without port, the default port is 443

Reply:
9200, is es's default port, use 127.0.0.1 can't call the api, it's a config error?

Reply:
Ok, thanks for reviewing the code

Reply:
![image](https://github.com/apache/linkis/assets/125547374/485509bb-ffcd-4e5a-9b49-1d37fcec9f16)
![image](https://github.com/apache/linkis/assets/125547374/81f8bb4a-ee27-4a27-ac12-e6f1b1bf2222)

I did this because I wanted to be consistent with this configuration